config/initializers/colorized_logger.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+if Rails.env.local?
+  # This initializer adds color to the Rails logger output. It's a nice way to
+  # visually distinguish log levels.
+  module ColorizedLogger
+    COLOR_CODES = {
+      debug: "\e[36m", # Cyan
+      info: "\e[32m",  # Green
+      warn: "\e[33m",  # Yellow
+      error: "\e[31m",  # Red
+      fatal: "\e[35m",  # Magenta
+      unknown: "\e[37m" # White (or terminal default)
+    }.freeze
+
+    RESET = "\e[0m"
```

Comment:
```suggestion
    RESET = "\e[0m".freeze
```

Reply:
strings are frozen by default because of the magic comment at the top of the file
```
# frozen_string_literal: true
```